website/public/js/controllers/sortableInventoryCtrl.js

```diff
@@ -0,0 +1,14 @@
+habitrpg.controller('SortableInventoryController', ['$scope',
+  function ($scope) {
+
+    // doing this in a function causes 10 $digest iteration error
+    $scope.attributeSort = {};
+    $scope.attributeSort[env.t('con')] = ['-con', '-(con+int+per+str)'];
```

Comment:
Yea, you're right. I have also been thinking about testing the jade file and I think that there's just too much logic in there.  The sort key => sort order is split between the controller and view.  I should probably move all of that to the controller where it can be more easily tested.